### PR TITLE
update privacy and policy

### PR DESCRIPTION
**Pull request by AnishMishra1**

### CODE REVIEW

File: src/App.tsx
Comment (sachin691) [inline src/App.tsx:1]: Keep the code formatted, avoid extra spacings, lines etc.

File: src/globalAssets/22211.jpg
Comment (sachin691) [inline src/globalAssets/22211.jpg:1]: remove this image.

File: src/components/Privacy/assets/loginBg.jpg
Comment (sachin691) [inline src/components/Privacy/assets/loginBg.jpg:1]: I dont think this image is used in somewhere in privacy, I think you can remove the complete assets folder from privacy.

File: src/App.tsx
Comment (sachin691) [inline src/App.tsx:1]: I think you have used only subComponents, follow the structure followed in the repo.
Eg. create main file Privacy.tsx in components/privacy folder and then create a subComponent folder in which you will create files which will be used in Privacy.tsx.
walk through the other components, you'll understand the structure.

**Review by sachin691 — COMMENTED** [review]
Please complete the given requested changes.

File: src/App.tsx
Comment (AnishMishra1) [reply src/App.tsx:1]: i resolve the problem by removing space and line

File: src/globalAssets/22211.jpg
Comment (AnishMishra1) [reply src/globalAssets/22211.jpg:1]: image is removed.

File: src/components/Privacy/assets/loginBg.jpg
Comment (AnishMishra1) [reply src/components/Privacy/assets/loginBg.jpg:1]: remove the asset folder done.

File: src/App.tsx
Comment (AnishMishra1) [reply src/App.tsx:1]: resolved by understanding the struture of previous components

**Review by ruck-45 — APPROVED** [review]

**Pull request merged**